package.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
Any reason for deleting this file?

Reply:
i didn't delete it.. maybe cause i installed a new package